### PR TITLE
Creation of the skeleton.

### PR DESCRIPTION
**Pull request by NicolasStr**
Let's start programming now.
Next PR will be adding CI/CD

### CODE REVIEW

File: src/layout/Footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    className="fa fa-heart"
+                    style={{ margin: "5px" }}
+                  />
+                  by <a>TypeCobol Team</a>
+                </p>
+              </Column>
+            </Columns>
+            <Content isSize="small">
+              <p>
+                The source code is licensed under of the TypeCobol project is
```
Comment (smedilol) [inline src/layout/Footer/index.tsx]: Typo I think: "The source code is licensed under of " can be removed ?

File: src/layout/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                licensed under
+                <a target="_blank">CeCILL-C</a>.
+              </p>
+              <p>
+                The website content is licensed under
+                <a target="_blank">CC ANS 4.0</a>.
```
Comment (smedilol) [inline src/layout/Footer/index.tsx]: We cannnot use this licence without checking with our jurist department.
We better keep CeCILL-C to be sure.

**Review by smedilol — DISMISSED** [review]

File: src/layout/Footer/index.tsx (hunk 1 above)
Comment (NicolasStr) [reply src/layout/Footer/index.tsx]: Resolved in another commit

File: src/layout/Footer/index.tsx (hunk 2 above)
Comment (NicolasStr) [reply src/layout/Footer/index.tsx]: Resolved in another commit

**Comment by NicolasStr** [comment]
I'm closing this Pull Request, I just updated too much and will create a new PR with all of these commits.

**Pull request closed without merging**